packages/repository-tests/src/__tests__/acceptance/default-repository.memory.acceptance.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
(Out of the scope of this PR) Would it make more sense if move the memory acceptance test to the [acceptance dir](https://github.com/strongloop/loopback-next/tree/f1de448ec886c5dca7eb86f019d2fd87048b4565/acceptance) for the sake of consistency?

Reply:
> Would it make more sense if move the memory acceptance test to the acceptance dir for the sake of consistency?

No.

I am intentionally keeping memory-based tests inside `packages/repository-tests` to ensure we execute the shared test suite as part of regular `npm test` and that the coverage data include lines covered by the shared test suite.

Here is the typical workflow:
- When making changes in loopback-next, we run the shared test suite using the memory connector, because it's relatively fast.
- We don't run the tests again databases, because it's expensive and the probability of breaking functionality in a database-specific way is very very low.
- However, we still want to verify that our code works with real databases, that's why we have dedicated Travis CI jobs running these tests for pull requests.
- In the rare case that we broke some of the shared tests with one or more databases, we can run the tests in `acceptance/repository-{database}` manually, as described in READMEs.

Reply:
👍 I see fair enough. And I like running those acceptance tests separately.